README.md

```diff
@@ -162,6 +163,94 @@ Hydrogen implements the [messaging protocol](http://ipython.org/ipython-doc/stab
 - In order to have access to your `$PATH` to find where IPython and other binaries are, Atom has to be launched from the command line with `atom <location>`. If you launch Atom as an app, this package won't work.
 
 
+## Custom kernel connection (inside Docker)
+
+You can use a custom kernel connection file to connect to a previously created kernel.
+
+For example, you can run a kernel inside a Docker container and make Hydrogen connect to it automatically. If you are using Docker this would allow you to develop from Atom but with all the dependencies, autocompletion, environment, etc of a Docker container.
+
+Hydrogen will look for a kernel JSON connection file under `./hydrogen/kernel/hydrogen-kernel.json` inside your project. If that file exists, Hydrogen will try to connect to the kernel specified by that connection file.
+
+Here's a simple recipe for doing and testing that with Python:
+
+* In your project directory, create a `Dockerfile` with:
+
+```
+FROM python:2.7
+
+RUN pip install markdown
+
+RUN pip install ipykernel
+RUN echo "alias hydrokernel='python -m ipykernel "'--ip=$(hostname -I)'" -f /tmp/kernel/hydrogen-kernel.json'" >> /etc/bash.bashrc
+```
+
+You will test using the Python package `markdown` from inside the Docker container in your local Atom editor, with autocompletion, etc.
+
+The last two lines are the only (temporal) addition to your `Dockerfile` that will allow you to develop locally using the remote Python kernel. If you already have a Python project with a `Dockerfile` you only need to copy those 2 lines and add them to it:
+
+```
+RUN pip install ipykernel
+RUN echo "alias hydrokernel='python -m ipykernel "'--ip=$(hostname -I)'" -f /tmp/kernel/hydrogen-kernel.json'" >> /etc/bash.bashrc
```

Comment:
Clever use of single and double quotes!

Wouldn't it make more sense to run the command `python -m ipykernel --ip=$(hostname -I) -f /tmp/kernel/hydrogen-kernel.json` from .bashrc rather than define an alias.